pyabc/population.py

```diff
@@ -196,8 +196,8 @@ def get_weighted_distances(self) -> pandas.DataFrame:
         Returns
         -------
 
-        weighted_distances: pandas.DataFrame:
-            A pandas.DataFrame containing in column 'distance' the distances
+        weighted_distances: pd.DataFrame:
+            A pd.DataFrame containing in column 'distance' the distances
```

Comment:
I would write "A pandas DataFrame ... "
The abbreviation of `pd` is widely used, but it is not the exact name of the package

Reply:
good point. I used pd here now cause I had changed the import to ``import pandas as pd``. Best would be to generate links to the respective classes here ...